### PR TITLE
Phenov2

### PR DESCRIPTION
**Pull request by gsfk**
upgrade / refactor:
- change output to phenopackets v2
- replace most uniform distributions with gaussian or exponential (this will make overviews look a bit less weird)
- make the data more variable, so that it's more like prod data, and more likely to uncover bugs in testing
- provide a mechanism for adding arbitrary real experiment files (previously hardcoded for 1000 genomes only)
- add bigwigs 
- add miscellaneous experiment files (.csv, .docx, .jpg, .md, .mp4, .pdf)
- provide matching katsu config and DATS files

### CODE REVIEW

File: requirements.txt
[the remark this answers is not in the document]
Comment (gsfk) [reply requirements.txt:1]: whoops, thanks

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,9 @@
-Faker==8.8.2
-numpy==1.22.0
-python-dateutil==2.8.1
-six==1.13.0
-text-unidecode==1.3
+Faker           22.2.0
+numpy           1.26.2
+pip             23.3.1
+pkg_resources   0.0.0
```
[the remark this answers is not in the document]
Comment (gsfk) [reply requirements.txt:4]: huh, yeah, this is an [ubuntu bug](https://bugs.launchpad.net/ubuntu/+source/python-pip/+bug/1635463)

File: dataset_files/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+## -Omics files
+
+
+**VCFs**: See [1000 Genomes on GRCh38](https://www.internationalgenome.org/data-portal/data-collection/grch38) for vcf data. Files are expected to be divided by sample rather than by chromosome. Do this yourself, or run [this otherwise deprecated script](../sample.vcf.sh) to produce 1000 genomes vcf files divided by sample, but covering chromosome 22 only (in GRCh37).
```
[the remark this answers is not in the document]
Comment (gsfk) [reply dataset_files/README.md:4]: We could do this at some point. I'm not fully satisfied with the current approach ([1000 Genomes on GRCh38](https://www.internationalgenome.org/data-portal/data-collection/grch38) lacks a Y chromosome) but it's not immediately clear that other options are better. To be discussed.

**Review by v-rocheleau — APPROVED** [review]
Lgtm!

**Pull request merged**